be/src/app.ts

```diff
@@ -5,13 +5,13 @@ import RootRouter from './controlers/root.router';
 import RestRouter from './controlers/rest.router';
 import ResolveRouter from './controlers/resolve.router';
 import HealthRouter from './controlers/health.router';
+import * as bodyParser from "body-parser";
 // @ts-ignore
 const express = require('express');
 
 const serverConfig: ServerConfig = new ServerConfig();
 const app: Application = express();
-
-// app.use(json);
+app.use(bodyParser.json())
```

Comment:
Please try the out of the box included `express.json()` from express, see https://stackoverflow.com/a/62396576

Reply:
Hmm. We tried, but slightly different...

@dan0bar - could you try this variant?
```suggestion
app.use(express.json())
```

Reply:
Doesn't work either...
TypeError: express(...).json is not a function

Reply:
It's `express.json()` not `express().json`

Small but significant difference.